service-commands/scripts/userClockValues.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Find the creator nodes associated with a user and their respective clock values.
+ *
+ * Script usage: node userClockValues.js <size> <outFile>
+ */
+const axios = require('axios')
+const CreatorNode = require('@audius/libs/src/services/creatorNode')
+const { Command } = require('commander')
+
+function commaSeparatedList(value, unusedPrevValue) {
+  return value.split(',')
+}
+
+const program = new Command()
+program
+  .usage('')
```

Comment:
if we can bake all this into an existing cli somehow instead of writing scripts for each of these functions that would be 💯

Reply:
Yes, that makes sense, I'll open another PR for moving this and the white noise script somewhere else so that it's more maintainable.